tsdb/head.go

```diff
@@ -753,7 +758,7 @@ func (h *Head) truncateMemory(mint int64) (err error) {
 	start := time.Now()
 
 	actualMint := h.gc()
-	level.Info(h.logger).Log("msg", "Head GC completed", "duration", time.Since(start))
+	level.Debug(h.logger).Log("msg", "Head GC completed", "duration", time.Since(start))
```

Comment:
This should be `Info`, no need to change. It is logged only once every 2 hrs and a useful log to have.

Reply:
oops, I misread https://github.com/prometheus/prometheus/issues/5599#issuecomment-623483518, reverting.